### PR TITLE
Bug/custom loader context

### PR DESCRIPTION
**Pull request by jonsherrard**

### CODE REVIEW

**Comment by jonsherrard** [comment]
[Bug: Custom Loaders should have full loadContext object passed as a parameter](https://trello.com/c/mgVM38mA/41-bug-custom-loaders-should-have-full-loadcontext-object-passed-as-a-parameter)

File: src/utils/location-origin-polyfill.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default function locationOriginPolyfill() {
```
Comment (thomasdigby) [inline src/utils/location-origin-polyfill.js]: Can we include the polyfill as a npm package?

File: src/loader-post.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ export default class Loader extends Component {
 
     console.log(loadContext)
     const customLoader = loadContext.loaders.Post
```
Comment (thomasdigby) [inline src/loader-post.js]: I think we need to detect whether `loaders` have been declared on `loadContext` before we access `Post`. If you remove `loaders` from `tapestry.js` config, it'll break. My bad. 🐙

File: src/utils/location-origin-polyfill.js (hunk 1 above)
Comment (jonsherrard) [inline src/utils/location-origin-polyfill.js]: Good point... definitely not the first person to have had this problem!

File: src/loader-post.js (hunk 2 above)
Comment (jonsherrard) [inline src/loader-post.js]: Good spot. Will update.

**Pull request merged**